src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/10 Page/evaluateHandle.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 'evaluateHandle(pageFunction, [arg])'
+excerpt: 'Browser module: page.evaluateHandle(pageFunction, [arg]) method'
+---
+
+Returns the value of the `pageFunction` invocation as a [JSHandle](/javascript-api/k6-experimental/browser/jshandle/). 
+
+The only difference between `page.evaluate` and `page.evaluateHandle` is that `page.evaluateHandle` returns [JSHandle](/javascript-api/k6-experimental/browser/jshandle/).
+
```

Comment:
We should probably have a _Returns_ sub heading in here.

Reply:
Resolve in https://github.com/grafana/k6-docs/commit/95398b184f9afe52df89593693d16dfa2d7085c5